packages/preview-middleware/templates/flp/sandbox2.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<!-- Copyright (c) 2015 SAP AG, All Rights Reserved -->
+
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Local FLP Sandbox</title>
+
+    <!-- Bootstrap the unified shell in sandbox mode for standalone usage.
+
+         The renderer is specified in the global Unified Shell configuration object "sap-ushell-config".
+
+         The fiori2 renderer will render the shell header allowing, for instance,
+         testing of additional application setting buttons.
+
+         The navigation target resolution service is configured in a way that the empty URL hash is
+         resolved to our own application.
+
+         This example uses relative path references for the SAPUI5 resources and test-resources;
+         it might be necessary to adapt them depending on the target runtime platform.
+         The sandbox platform is restricted to development or demo use cases and must NOT be used
+         for productive scenarios.
+    -->
+    <script type="text/javascript">
+        window["sap-ushell-config"] = {
+            defaultRenderer: "fiori2",
+            renderers: {
+                fiori2: {
+                    componentData: {
+                        config: {
+                            search: "hidden",
+                            enableSearch: false
+                        }
+                    }
+                }
+            },
+            applications:  <%- JSON.stringify(apps) %>
+        };
+    </script>
+
+    <script type="text/javascript">
+        window["data-open-ux-preview-basePath"] = "<%- basePath %>";
+    </script>
+
+    <script src="<%- basePath %>/resources/sap/ushell/bootstrap/sandbox2.js" id="sap-ushell-bootstrap"></script>
+    <!-- Bootstrap the UI5 core library. 'data-sap-ui-frameOptions="allow"' is a NON-SECURE setting for test environments -->
+    <script id="sap-ui-bootstrap"
+        src="<%- basePath %>/resources/sap-ui-core.js"
+        data-sap-ui-libs="<%- ui5.libs %>"
+        data-sap-ui-async="true"
+        data-sap-ui-preload="async"
+        data-sap-ui-theme="<%- ui5.theme %>"
+        data-sap-ui-compat-version="edge"
+        data-sap-ui-language="en"
+        data-sap-ui-bindingSyntax="complex"
```

Comment:
What about `bindingSyntax`, `frameOptions`, `componentPreload` and `flexibilityServices`? Will they stay camelCase with 2.0 or also change?

@petermuessig

Reply:
Valid point. At least `frame-options` and `flexibility-services` are listed in the [documentation](https://sapui5.hana.ondemand.com/sdk/#/topic/91f2d03b6f4d1014b6dd926db0e91070). Not sure about `bindingSyntax` and `componentPreload` though. I'll eject the file and see what the linter has to say about it.

Reply:
Hm. Maybe `data-sap-ui-bindingSyntax` and `data-sap-ui-preload` are no longer valid at all? 🤔 
![image](https://github.com/user-attachments/assets/67b8c9c1-1b8a-4ba6-b638-821ccf210072)

No information on `data-sap-ui-xx-componentPreload` from the linter 🤷🏻

Reply:
There seem to be some more parameters (found in the test snaps):
- `data-sap-ui-xx-designMode`
- `data-sap-ui-xx-suppressDeactivationOfControllerCode`
- `data-sap-ui-xx-viewCache`

Reply:
@matz3 can you by any chance comment on the UI5 bootstrap parameter names?

Reply:
`sap-ui-compat-version="edge"` should be set. It also sets the `bindingSyntax` to `complex` internally, so no need to provide that parameter additionally.

I don't know the status about `sap-ui-xx-componentPreload` / `sap-ui-xx-viewCache`. Better double-check this with the core framework team. That's also the reason why UI5 linter currently doesn't show a finding.

`sap-ui-xx-designMode` and `sap-ui-xx-suppressDeactivationOfControllerCode` are restricted parameters that must only be used by the listed consumers (e.g. see https://github.com/SAP/openui5/blob/master/src/sap.ui.core/src/sap/ui/base/DesignTime.js)

Reply:
`data-sap-ui-preload` seems to be replaced by `data-sap-ui-async="true"` since UI5 quite some time (see [documentation of UI5 1.71.57](https://ui5.sap.com/1.71.57/#/topic/91f2d03b6f4d1014b6dd926db0e91070)

Reply:
`sap-ui-xx-designMode` and `sap-ui-xx-suppressDeactivationOfControllerCode` are allowed for RTA so this should be fine, just need to check the naming if its still `camelCase` or if `sap-ui-xx-` switched to `kebap-case` as well.

@codeworrior can you comment on the naming convention of the `sap-ui-xx-` properties in UI5 2.x?

Reply:
As HTML5 does not support case-sensitive attribute names (DOM APIs return attribute names all lower case), the new sap/base/config enabled kebab-case everywhere. The old names (all lower case, created from camelCase in HTML source) are still understood, but I would recommend to switch to the kebab-case, if the UI5 version can be assumed >= 1.120.